### DISCUSSION TITLE
What .svg object or .kvconfig line controls the view background in Dolphin?

### DISCUSSION THREAD

**Discussion by salperinlea** [tsujan/Kvantum#825] [Q&A]
I am trying to set the color of the viewing background (namely, the background behind folders, files, etc.) in dolphin. My efforts continue to fail.

If one sets application style to Breeze, the color specified as "View Background" will be used for the area in question; if the application style is set to kvantum, however, this color will not be used. So I figure that there must be something in Kvantum's .kvconfig and/or svg that sets that element. I know what color it's set to now, and if I I change EVERY element that has that color to my desired one, this element won't be changed.

I could really use a pointer about what to do. 
I've attached screenshots of Dolphin in kvantum and breeze app styles to illustrate the difference I mean.

Kvantum:
![kvantum_dolphin](https://user-images.githubusercontent.com/40773827/194121416-7153f04e-40ff-494e-9844-d79369f59edc.png)

Breeze:
![breeze_dolphin](https://user-images.githubusercontent.com/40773827/194121232-7dd3f671-5cc7-4c0f-9f5d-6b9c5c86dc5e.png)

**Comment by tsujan** [comment]
It's `base.color` and isn't specific to Dolphin. Don't forget to uncheck `Kvantum Manager → Configure Active Theme → Transparent Dolphin view` (it's unchecked by default, but some themes check it).

> If one sets application style to Breeze, the color specified as "View Background" will be used for the area in question; if the application style is set to kvantum, however, this color will not be used

KDE color scheme is for KDE styles. Kvantum has its own colors.
